src/cmd/run.js

```diff
@@ -123,7 +124,13 @@ export default async function run(
 
   const profileDir = firefoxProfile || chromiumProfile;
 
-  if (profileCreateIfMissing && profileDir) {
+  if (profileCreateIfMissing) {
+    if (!profileDir) {
+      throw new UsageError(
+        '--profile-create-if-missing should be paired ' +
```

Comment:
would you mind to replace `should be paired with` with `requires`? (mostly because I'm not really sure if paired is a common why to refer to that in english, "--profile-create-if-missing requires --firefox-profile or --chromium-profile" is shorter and should be clear enough).

Same change has to be applied to the assertion on the test side.

Reply:
> @akhilpanchal good job, this looks great. I would just tweak the UsageError message a bit (my apologies I should have noticed it sooner and pointed it out in the previous review round) before merging this on master, would you mind to do that? (the tweak is described below).

No worries at all! Makes sense to me. It is shorter and clear.
I addressed it.